src/Term.ts

```diff
@@ -1,17 +1,33 @@
 import { exec } from "@actions/exec";
+import hasYarn from "has-yarn";
 
-class Git {
+const INSTALL_STEP = "install";
+const BUILD_STEP = "build";
+
+class Term {
   async execSizeLimit(
-    branch?: string
+    branch?: string,
+    skipStep?: string
   ): Promise<{ status: number; output: string }> {
     let output = "";
 
     if (branch) {
       await exec(`git checkout -f ${branch}`);
     }
 
-    await exec("npm install");
-    await exec("npm run size-build");
+    if (skipStep !== INSTALL_STEP && skipStep !== BUILD_STEP) {
+      try {
+        await exec("npm run size-install");
```

Comment:
In my opinon it is an interesting approach to have a custom npm script instead of picking up on settings in `package.json` or an rc file (as other tools [like size-limit itself](https://github.com/ai/size-limit#config)).

Reply:
I just thought that would be easier but I think you are right, `inputs` seems like the best approach. I'll merge this PR for now and create an issue to this discuss this further.